README.md

```diff
@@ -177,4 +178,33 @@ After successfully deploying the infrastructure with this module, you'll need to
 1. Deploy your first Materialize environment
 
 See our [Operator Installation Guide](docs/operator-setup.md) for instructions.
+
+## Connecting to Materialize instances
+
+By default, Network Load Balancers are created for each Materialize instance, with three listeners:
+1. Port 6875 for SQL connections to the database.
+1. Port 6876 for HTTP(S) connections to the database.
+1. Port 8080 for HTTP(S) connections to the web console.
+
+The DNS name and ARN for the NLBs will be in the `terraform output` as `nlb_details`.
```

Comment:
More of a question to ensure I understand what's happening.  If I have the following: 
```
[
  {
    "arn" = "..."
    "dns_name" = "...."
  },
]
```
Should I be able to connect a psql to the dns_name like the following?
`psql -h <dns> -p 6875`

Reply:
Thanks for the clarification!